client/components/Editors/Table/columns.tsx

```diff
@@ -40,74 +41,85 @@ export function createColumns(
   const dataColumns: IColumn[] = []
   const dataFields = getDataFields({ schema, report })
   for (const field of dataFields) {
-    let header = field.title ?? field.name
-    const errors = errorIndex.label[field.name]
-    if (errors) {
-      const error = errors[0]
-      // @ts-ignore
-      if (error) header = error.label
+    const labelErrors = errorIndex.label[field.name]
+
+    const renderHeader: IColumn['header'] = () => {
+      const firstError = labelErrors?.[0]
+      const label = firstError ? firstError.label : field.title ?? field.name
+
+      if (firstError) {
+        return (
+          <LightTooltip title={firstError.message}>
+            <Box sx={{ width: '80%', minHeight: '20px' }}>{label}</Box>
+          </LightTooltip>
+        )
+      }
+
+      return label
     }
+
+    const renderRow: IColumn['render'] = (context) => {
+      const { cellProps, data } = context
+      let { value } = context
+      const rowNumber = data._rowNumber
+      const columnName = cellProps.id
+      const rowKey = `${rowNumber}`
+      const cellKey = `${rowNumber},${columnName}`
+
+      // Selection
+      if (selection) {
```

Comment:
I think this is not being used anywhere @roll  . It's not part of this ticket, but i think we should delete this if. Or in which case is this used? I tested and couldn't see it.

Reply:
I think you're right but just in case I think it's better to remove obsolete code as a separate PR (there are also a lot not used dialogs etc)